docs/develop/typescript/index.mdx

```diff
@@ -25,7 +26,7 @@ Build Temporal Applications with the TypeScript SDK.
 
 - [Temporal TypeScript Community Slack](https://temporalio.slack.com/archives/C01DKSMU94L)
 - [TypeScript SDK Forum](https://community.temporal.io/tag/typescript-sdk)
-:::
+  :::
```

Comment:
Same issue here. Please review all the pages for the admonition styling (::: .... ::::)